setup.py

```diff
@@ -11,7 +11,7 @@ def main():
 
     extras = {
         'dev': [
-            'pylint',
+            'pylint==2.9.6',
```

Comment:
Maybe even `>=2.9.6`?

Reply:
I'd rather not (at least: not now), see https://github.com/neptune-ai/neptune-fastai/pull/27#discussion_r870996558